### DISCUSSION TITLE
[x][closed] Preferred name for H3K4me3 in SO ontology it H3K4_trimethylation. Should we enforce this?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#85] [Team Posts]
It is more common to use H3K4me3 in my experience. However, allowing both creates confusion in a value selection list. Or perhaps use another ontology?

**Comment by sveinugu** [comment]
José María's answer: If the ontology term has H3K4me3 in its synonyms, we can still use it. But we can also look for other ontologies.

**Comment by sveinugu** [comment]
Daniel's answer: IHEC is also struggling with this. I'm not convinced SO is a particularly relevant authority in this matter.

**Comment by sveinugu** [comment]
@jmfernandez So my idea with having the "term_value" attribute alongside the "term_iri" was that one could in tools just simply list the term_value strings instead of having to look up every term in the ontology. However, if we are not stringent we will end up with lists that contains "H3K4me3", "H3K4_trimethylation", and other variations of the same, which is exactly the issue I would like us to fix. So having strict validation towards the preferred name would solve this. But in this case, the preferred name of the ontology was not my preferred name..

**Comment by sveinugu** [comment]
@dzerbino I could not find any better authority than SO. Seems to me to cover most types of tracks. The focus is to cover types of annotation features, but a tracks is nothing more than a set of annotation features, most often of the same type. But do you have a suggestion of a more relevant authority?

**Comment by dzerbino** [comment]
IHEC are developing their controlled vocab, and SO was not a contender. Frankly _trimethylation sounds very old fashioned to me.

**Comment by sveinugu** [comment]
Conclusion (for now): we use SO
